test/download.spec.js

```diff
@@ -83,6 +83,19 @@ describe('Download', function() {
     expect(fs.readFileSync(path).toString()).toBe('Hello world');
     await page.close();
   })
+  it(`should report download path within page.on('download', …) handler for Blobs`, async({browser, server}) => {
```

Comment:
```suggestion
  it.fail(FFOX || WEBKIT)(`should report download path within page.on('download', …) handler for Blobs`, async({browser, server}) => {
```

@pavelfeldman Should I add a `fail` scoped to the browsers that I know this will fail on right now?

Reply:
@rwoll yes, please add .fail() for the browsers where it is failing.